website/addons/base/views.py

```diff
@@ -203,6 +204,21 @@ def create_waterbutler_log(payload, **kwargs):
     node = kwargs['node'] or kwargs['project']
 
     if action in (NodeLog.FILE_MOVED, NodeLog.FILE_COPIED):
+        dest = payload['destination']
+        src = payload['source']
+
+        if src is not None and dest is not None:
+            dest_path = dest['materialized']
+            src_path = src['materialized']
+            if str(dest_path).endswith("/") and str(src_path).endswith("/"):
+                dest_path = os.path.dirname(dest_path)
+                src_path = os.path.dirname(src_path)
+            if os.path.split(dest_path)[0] == os.path.split(src_path)[0]:
+                if dest['provider'] == src['provider']:
```

Comment:
You'd be better off using `all` here.
```python
dest['materialized'].replace(dest['name']) == src['materialized'].replace(src['name'])
```
The above should be a good way to see if just the name was changed. But you'll need function that will do a right to left string replacement limited to a single iteration.